server/sonar-server/src/main/java/org/sonar/server/component/index/ComponentIndexQuery.java

```diff
@@ -29,12 +29,18 @@
 
 public class ComponentIndexQuery {
 
+  public enum Sort {
+    NAME, SCORE
+  }
```

Comment:
After offline discussion, I won't add specific behavior in the Sort in order to not have dependency on ES, but I'll use a switch when getting sort in order to fail when a new value has been added that has not been handled